packages/@aws-cdk/aws-route53/test/test.hosted-zone-provider.ts

```diff
@@ -1,47 +1,85 @@
 import { SynthUtils } from '@aws-cdk/assert';
 import cdk = require('@aws-cdk/core');
 import { Test } from 'nodeunit';
-import { HostedZone, HostedZoneAttributes } from '../lib';
+import { HostedZone } from '../lib';
 
 export = {
   'Hosted Zone Provider': {
-    'HostedZoneProvider will return context values if availble'(test: Test) {
+    'HostedZoneProvider will return context values if available'(test: Test) {
       // GIVEN
-      const stack = new cdk.Stack(undefined, 'TestStack', { env: { account: '12345', region: 'us-east-1' } });
-      const filter = {domainName: 'test.com'};
+      const stack = new cdk.Stack(undefined, 'TestStack', {
+        env: { account: '12345', region: 'us-east-1' }
+      });
+      const filter = { domainName: 'test.com' };
```

Comment:
To improve code reviewer experience, try to avoid formatting changes (it's hard to ensure that there are no functional changes here)